src/components/BulkMetadataForm/__stubs__/uischema.ts

```diff
@@ -0,0 +1,20 @@
+import { MetadataProperties as fields } from '../../../types';
```

Comment:
Should this not be used in production as well? In that case a stub is not the right shape for this.